pkg/kubectl/sorting_printer.go

```diff
@@ -153,11 +155,56 @@ func isLess(i, j reflect.Value) (bool, error) {
 		return i.String() < j.String(), nil
 	case reflect.Ptr:
 		return isLess(i.Elem(), j.Elem())
+	case reflect.Struct:
+		// special case handling, unversioned.Time(TODO: add more)
```

Comment:
remove the TODO, since there is notthing to be done. If the need arises, it will happen:)